### PR TITLE
Migration cleanup

### PR DESCRIPTION
**Pull request by apinnick**
Cleaning up bits and pieces

Cherrypick for 4.2 and 4.3

### CODE REVIEW

File: modules/migration-installing-cam-operator-ocp-4.adoc [hunk 1]
```diff
@@ -8,66 +8,44 @@ ifdef::sourcecluster-4_1-4_x[]
...
 
 You can install the Cluster Application Migration Operator on an {product-title} {product-version} target cluster with OLM.
 
-The Cluster Application Migration Operator installs the following on the target cluster:
-
-* Migration controller CR
-* CAM web console
-* Restic
-* Velero
+The Cluster Application Migration Operator installs the CAM tool on the target cluster by default. If you want to install the CAM tool on a different cluster, you must update the Migration controller configuration so that Cluster Application Migration Operator does not install the CAM tool on this cluster.
```
Comment (bergerhoffer) [inline modules/migration-installing-cam-operator-ocp-4.adoc]: Needs a "the" in front of the second instances of "Cluster Application Migration Operator"

**Review by bergerhoffer — COMMENTED** [review]
Just one minor thing, otherwise LGTM!

**Comment by apinnick** [comment]
@bergerhoffer 
Fixed! Thanks

**Comment by bergerhoffer** [comment]
LGTM!

**Pull request merged**